pages/api/drawer.md

```diff
@@ -6,7 +6,8 @@ filename: /src/Drawer/Drawer.js
 
 # Drawer
 
-
+The properties of the [Modal](/api/modal) component are also available
+when `variant="temporary"`.
```

Comment:
This reads a bit weird, maybe change it to "...also available if `variant` is set to `"temporary"`."